great-idea/css/index.css

```diff
@@ -63,4 +63,263 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+ol, ul {
+	list-style: circle;
+}
+
+.main-content {
+	margin: 0 25%;
+}
+
+header {
+	display: flex;
+	margin: 5% 5% 5% 0;
+}
+
+nav {
+	display: flex;
+	justify-content: space-evenly;
+	width: 80%;
+}
+
+nav a {
+	margin-top: 1%;
+	text-decoration: none;
+	color: grey;
+}
+
+/* .logo {
+	margin-right: 3%;
+} */
+
+.top-feature {
+	display: flex;
+	justify-content: space-evenly;
+	margin: 5% 10%;
+}
+
+.feature-text {
+	width: 30%;
+	height: 220px;
+	margin: 5% 10%;
+}
+
+.feature-text h2 {
+	font-size: 70px;
+	display: flex;
+	justify-content: center;
+}
+
+.get-started-btn {
+	margin-left: 25%;
+	width: 50%;
+	height: 15%;
+	background-color: #fff;
+}
+
+.top-content {
+	display: flex;
+	margin: 4% 0;
+	justify-content: space-around;
+}
+
+.features {
+	width: 45%;
+}
+
+.about {
+	width: 45%;
+}
+
+.middle-image {
+	display: flex;
+	justify-content: center;
+}
```

Comment:
The middle image seems to work more or less like it should in Firefox, but obstinately stays the same size no matter what in Chrome, (_in my computer at least_). There seems to be better behavior if you make the width of both the image and the div which encloses to 100%.

In addition, `justify-content: center;` would center the image inside the  `.middle-image` div, but it won't center the div itself, so if there's a situation that the image needs to be centered on the page, make sure the div itself is centered. This is not an issue here, because the image is meant to take the whole width of the `.main-content` container sans margin.

Speaking of margins, the image doesn't seem to be lined up with the `.top-content` and `.bottom-content` divs. You can mess with the left/right margins on the `.middle-image` div to make it jive better. Another approach is to make both content divs `justify-content: space-between;`, then add consistent left/right margins to `.top-content`, `.bottom-content`, and `.middle-image`.

---

Side opinionated note: I suggested earlier to make the `.middle-image` div and its `.middle-img` image child's width to 100%. I also noted that if the image needed to centered in any way, the image should be centered in the div, and div should be centered in relation to the surrounding elements. Notice that we need to juggle _two_ elements to style one component on the screen.

It might make more sense to fire that surrounding div on the image and style the image directly:
```
.middle-img /*the class name of the image*/ {
   width: 100%;
}
```
Versus:
```
.middle-image {
  width: 100%;
}

.middle-image img /* or .middle-img */ {
  width: 100%;
}
```

Reply:
As for the side opinion note, I was talking to some friends of mine, and they said there are use cases for having the `div` `img` combo.

This is why I prefaced that comment as a "side opinion", as it's more of what I think it should be, rather than something you _must_ do to get it right.